internal/testutils/manifestbuilder/manifestbuilder.go

```diff
@@ -97,6 +99,11 @@ func (b ManifestBuilder) WithLocalServeCmd(cmd string) ManifestBuilder {
 	return b
 }
 
+func (b ManifestBuilder) WithLocalDisableSource() ManifestBuilder {
```

Comment:
can you add some comments on when you'd want to call this and when you wouldn't want to call this? I don't totally understand why it's an option at all, rather than something that all Manifests get.

Reply:
the reason for this no longer exists and doing it for all manifests is good. thanks!